From 6ac631a90bc9d956c1666c18846a36f76887c64e Mon Sep 17 00:00:00 2001
From: Your Name <github-actions[bot]@users.noreply.github.com>
Date: Thu, 20 Nov 2025 15:33:42 +0800
Subject: [PATCH 01/34] chore: add local secrets and temp directory to
 .gitignore

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index e6422fa6..4b5e3818 100644
--- a/.gitignore
+++ b/.gitignore
@@ -56,3 +56,5 @@ supabase/.temp/
 **/.gradle/
 **/nb-cache/
 
+"supabase/secrets.json" 
+"supabase/.temp" 
-- 
2.51.2.windows.1

